urbit/umbrel-app.yml

```
@@ -35,10 +35,10 @@ torOnly: false
 submitter: ~mopfel-winrux
 submission: https://github.com/getumbrel/umbrel/pull/1246
 releaseNotes: >-
-  8GB loom has landed, which you can use by adding --loom 33 to your normal boot command. Additionally, comets can now only be booted with the -c flag, the strange page issue is fixed, --cue is added to urbit eval, and authentication requests are checked using eyre's tokens.
+  Arvo 414K
+  Vere v2.2

```

Extra newline so that these show on separate lines. Did you want `v2.2` to actually be `v2.1` here? I see in the github release info that it is `v2.2` but just double checking

```suggestion
  Arvo 414K
  
  Vere v2.2

```